src/StoreKeeper/WooCommerce/B2C/Imports/ProductImport.php

```diff
@@ -1039,11 +1039,74 @@ function ($attribute) {
                     return !key_exists('attribute_option_id', $attribute);
                 }
             );
-            foreach ($nonAttributeOptions as $attribute) {
+            // Find attributes that have 'attribute_option_color_hex'
+            $colorAttributes = array_filter(
+                $dotObject->get('flat_product.content_vars'),
+                function ($attribute) {
+                    return array_key_exists('attribute_option_color_hex', $attribute);
+                }
+            );
+
+            $mergedAttributes = array_merge($nonAttributeOptions, $colorAttributes);
+
+            $blocksyTaxonomyMetaOptions = [];
+
+            foreach ($mergedAttributes as $attribute) {
```

Comment:
you can make the loop only for the `$colorAttributes` instead of both `$nonAttributeOptions, $colorAttributes`